From b1cf509426fa30e450d734536eab8fad3bd6c4b4 Mon Sep 17 00:00:00 2001
From: Hariharan K <quic_harihk@quicinc.com>
Date: Thu, 10 Oct 2024 16:25:31 +0530
Subject: [PATCH] firmware: scm: Adding error code for image auth failure

Adding a new error code for denoting image authentication
failure during fuseipq call.

Change-Id: I3ae96e4a674b216e73bdb4c67c6affc01c29dbfd
Signed-off-by: Hariharan K <quic_harihk@quicinc.com>
---
 include/linux/firmware/qcom/qcom_scm.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/firmware/qcom/qcom_scm.h b/include/linux/firmware/qcom/qcom_scm.h
index 6e422f331386..9729870cf317 100644
--- a/include/linux/firmware/qcom/qcom_scm.h
+++ b/include/linux/firmware/qcom/qcom_scm.h
@@ -204,6 +204,7 @@ enum qcom_scm_ice_cipher {
 #define QCOM_SCM_SVC_FUSE          0x8
 #define FUSEPROV_INVALID_HASH      0x9
 #define FUSEPROV_SECDAT_LOCK_BLOWN 0xB
+#define IMAGE_AUTH_FAILURE         0x101
 #define QCOM_KERNEL_AUTH_CMD       0x15
 #define TZ_BLOW_FUSE_SECDAT        0x20
 #define QCOM_KERNEL_META_AUTH_CMD  0x23
-- 
2.34.1

